src/controllers/catalogue.controller.js

```diff
@@ -0,0 +1,58 @@
+import {
+  createCatalogueUseCase,
+  deleteCatalogueByIdUseCase,
+  getAllCataloguesQuery,
+  getCatalogueByIdQuery,
+  updateCatalogueUseCase
+} from '../repositories/catalogue.repository.js'
+
+export const getAllCatalogues = async (request, response) => {
```

Comment:
Please follow the convention names of the application using the req, res variables

Reply:
the req and res variables are code smells are not descriptive variables

Reply:
I let you the documentation of https://expressjs.com/es/guide/routing.html.

They are not code smells, just abbreviation, so you are telling me that using variable "args" in java is code smell?

Reply:
el papeador de papus